### ISSUE TITLE
Improve edit page accessibility

### ISSUE THREAD

**Issue by ChiuchiuSorin** [xwikisas/application-onlyoffice-connector#91]
There are several issues that can be improved:

1. Missing lang attribute on the HTML tag. Some screen readers can automatically switch languages based on the lang attribute.

2. iframe misses title attribute - An inline frame’s title provides a label for the frame that describes its content to the user. With this information a user can determine which frame to enter and explore in detail: https://www.w3.org/WAI/WCAG22/Techniques/html/H64

3. XWiki logo misses accessible name and link misses title

4. missing title on links

5. Missing landmarks for content
    - Landmark roles provide programmatic access to sections of a web page, making it easy for the user to understand the structure of the page and easily navigate to different sections: https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/
